create-a-theme-checklist.js: Also allow Github token from the environment

markbiek opened the pull request:
Create-a-theme-checklist can also get the Github token from the environment
    
In addition to hard coding a Github token in the actual `create-a-theme-checklist.js` file, we now have the option to read the token from the `THEME_GITHUB_TOKEN` environment variable.
    
This approach has the benefit of not including a secret like a token in source code.

matiasbenedetto reviewed (changes requested):
:wave: Thanks for submitting the change. Could you please remove the code style changes that are making the PR too verbose?

markbiek commented:
> 👋 Thanks for submitting the change. Could you please remove the code style changes that are making the PR too verbose?

@matiasbenedetto I'm happy to do it but I'm honestly not sure _when_ that formatting is happening. It seems like there may be a git hook that's doing the formatting when I push to Github.

I'll investigate and see what I can find.

In the meantime, if it helps, [2f349d1](https://github.com/Automattic/themes/pull/6206/commits/2f349d1ccc0a52848905551639751a0d0f8a9655) has the pertinent change.

markbiek committed:
Create-a-theme-checklist can also get the Github token from the environment

In addition to hard coding a Github token in the actual `create-a-theme-checklist.js` file, we now have the option to read the token from the `THEME_GITHUB_TOKEN` environment variable.

This approach has the benefit of not including a secret like a token in source code.

markbiek commented:
@matiasbenedetto Successfully updated so it's just the bare change, no formatting!

matiasbenedetto reviewed (approved):
Thanks.
LGTM
